### PR TITLE
Add the app icon navigation. Add tests.

### LINKED ISSUE #310: App icon leads to the main page

### PR DESCRIPTION
**Pull request by dmytroSolidSoftware**
Fixes #310

## Checklist

- [x] Documented
- [x] Unit tested

### CODE REVIEW

File: metrics/web/test/common/presentation/app_bar/widget/metrics_app_bar_test.dart [hunk 1]
```diff
@@ -2,10 +2,58 @@ import 'package:flutter/material.dart';
...
+      final finder = find.descendant(
+        of: find.byTooltip(CommonStrings.openUserMenu),
+        matching: find.byType(Image),
+      );
+
+      expect(finder, findsOneWidget);
+    });
+
+    testWidgets(
+      "navigates to the dashboard page on tap on the app logo",
```
Comment (bootstraponline) [inline metrics/web/test/common/presentation/app_bar/widget/metrics_app_bar_test.dart:40]:
```suggestion
      "navigates to the dashboard page when the app logo is tapped",
```

**Comment by vladlaba** [comment]
> I'm curious I think we should add the cursor change when user hovers over the app icon to provide visual feedback that it's palpable. What do you think?

Currently, there is no button within the Metrics app that supports changing the cursor. Therefore, and since we have a separate [issue](https://github.com/platform-platform/monorepo/issues/143) for that, I suppose that we should move this functionality to a separate PR.

**Comment by illia-romanenko** [comment]
Thanks, Vlad - I agree - let's fix it for all in the scope of the #143 - could you please work on it next?

**Comment by vladlaba** [comment]
> Thanks, Vlad - I agree - let's fix it for all in the scope of the #143 - could you please work on it next?

Thanks! Of course, we'll implement this 🚀

**Pull request merged**